### ISSUE TITLE
[WhaleHarness audit] hyhmrright/brooks-lint@1.5.0: REJECT

### ISSUE THREAD

**Issue by moby-whaleharness** [hyhmrright/brooks-lint#27]
WhaleHarness audited hyhmrright/brooks-lint@1.5.0 (commit d4b5c40804e6).

Verdict: REJECT
Reasons: x cordis.patch.yml missing; x dsh.bundle.patch must be ./cordis.patch.yml

Most rejections are fixable in minutes - a public fix guide for every reason:
https://whaleharness.com/audit-fixes.html

Verdicts are appealable and reproducible; one past false positive was corrected on appeal (Anionex/dsh-vision-toolkit#33). Re-submit anytime via https://whaleharness.com/submit.html

Moby @ WhaleHarness · moby@whaleharness.com

**Comment by hyhmrright** [comment]
This verdict applies a **profile-bundle** checklist to a repository that ships **Agent Skills**. The two are different DeepSeek Harness integration paths, and brooks-lint uses the latter.

**What brooks-lint actually is**

Flat markdown skills, installed by copying them into a skill root:

```sh
./scripts/install.sh dsh     # -> ${DSH_HOME:-~/.dsh}/skills
# or manually
cp -r skills/* ~/.dsh/skills/
```

Discovery is the standard one-level-deep `<root>/<name>/SKILL.md` layout (`.dsh/skills`, `.agents/skills`, `$DSH_HOME/skills`, `$DSH_AGENTS_HOME/skills`), documented in [`docs/dsh-setup.md`](https://github.com/hyhmrright/brooks-lint/blob/main/docs/dsh-setup.md). There is no runtime code, no Cordis plugin, and nothing published to npm as a bundle.

**Why the two REJECT reasons don't apply**

`dsh.bundle.patch` / `cordis.patch.yml` are the profile-bundle mechanism — `packages/bundle/base` in `deepseek-ai/deepseek-harness` states the package's substance *is* `cordis.patch.yml`, resolved by the profile composer through the `dsh.bundle.patch` manifest field. That is how a bundle patches the Cordis plugin graph.

A skills package patches nothing and loads no plugins, so:

- there is no manifest for `dsh.bundle.patch` to live in (`package.json` has no `dsh` key at all);
- a `cordis.patch.yml` would be dead weight with nothing to apply it.

Their absence is the correct state for this package type, not a defect. Verifiable at the audited commit `d4b5c40` — `git grep -iE 'cordis|dsh\.bundle|bundle\.patch'` returns zero matches, by design.

**Request**

Please either correct the verdict for this repo, or scope the bundle-manifest checks to packages that declare `dsh.bundle` and exclude skills-only repositories. Applying bundle requirements by topic match (`dsh-plugin` / `deepseek-harness`) will produce the same false positive on every skills package in the ecosystem.

Closing as not applicable. Happy to reopen if you can point to a DeepSeek Harness requirement that an Agent Skills package must ship a Cordis patch.

**Comment by moby-whaleharness** [comment]
更新 / Correction: WhaleHarness 的审计裁决已重构为分档制(安全红线/格式问题),
复核后 hyhmrright/brooks-lint@1.5.0 的当前判定是 **格式问题(format-issue)**。

此前通知中的旧判定(REJECT/RED-LINE 单档)已失效。若给作者带来困扰,我们道歉——
本轮重构起因于外部复审指出的误报,我们已修复判定逻辑并全量重扫。

最新数据: https://whaleharness.com/audit.json
修复指南: https://whaleharness.com/audit-fixes.html
申诉通道: https://github.com/WhaleHarness/WhaleHarness/discussions

Moby @ WhaleHarness

**Comment by moby-whaleharness** [comment]
更正 / Correction: 抱歉——复查发现 hyhmrright/brooks-lint 不是 DSH 插件(仅因 topic 打标被误抓进审计池),
此前关于它的审计判定已作废,不再适用。误抓是我们的对象选取缺陷,已修正分类。
打扰了。

Moby @ WhaleHarness
